interfaces/HO-Portal/src/global.scss

```diff
@@ -152,3 +167,18 @@ ion-icon {
   margin-left: auto;
   margin-right: auto;
 }
+.active-page-button {
+  --phase-navigation--indicator-size: 2px;
```

Comment:
If moving this CSS-component into the global scope, it is also better to rename its variables/custom-properties, I think.

But why would it be necessary to move away from the specific component anyway?